app/services/service_provider_updater.rb

```diff
@@ -1,15 +1,23 @@
 class ServiceProviderUpdater
   def run
     dashboard_service_providers.each do |service_provider|
-      service_provider = HashWithIndifferentAccess.new(service_provider)
-      issuer = service_provider['issuer']
-      SERVICE_PROVIDERS[issuer] = service_provider
-      VALID_SERVICE_PROVIDERS << issuer
+      update_local_caches(HashWithIndifferentAccess.new(service_provider))
     end
   end
 
   private
 
+  def update_local_caches(service_provider)
+    issuer = service_provider['issuer']
+    if service_provider['active'] == true
```

Comment:
My experience is no, the JSON serializer treats Ruby booleans as Javascript booleans. So it'll be a real boolean.

```
2.3.3 :001 > require 'json'
 => true 
2.3.3 :002 > payload = { active: true }
 => {:active=>true} 
2.3.3 :003 > payload.to_json
 => "{\"active\":true}" 
2.3.3 :004 > JSON.parse(payload.to_json)
 => {"active"=>true} 
```